src/facade/dragonfly_connection.cc

```diff
@@ -724,11 +725,29 @@ auto Connection::ParseMemcache() -> ParserStatus {
       break;
     }
 
+    auto parse_value = [&consumed, &cmd](std::string_view str) -> std::optional<std::string_view> {
+      std::string_view value = str.substr(consumed, cmd.bytes_len);
+      const absl::flat_hash_set<char> any_of{' ', '\r', '\n'};
```

Comment:
This is a small bug, I should add empty ' ' below and not here

Reply:
Kostas, thank for identifying the issue but I do not understand based on the code what exactly you are trying to check. Some style comments:
1. I do not think using sets for comparing with 3 ascii characters adds clarity. I think simple problems require simple solutions `if (c == '\r' || ...)`. You can also consider `base::_in(c, {...})`.
2. lambda captures are confusing on my opinion. why capture? why not passing as an argument?

see more comments below

Reply:
For (1) I don't really have a preference but I simplified the check so it's no longer needed anymore. For (2), I really don't see any problem with lambda captures but I will make the change since you find it confusing.